docs/tutorial/Basic/file_io.rst

```diff
@@ -78,6 +78,17 @@ Compared to the data structure of point cloud, mesh has triangles that define su
     Testing IO for meshes ...
     TriangleMesh with 1440 points and 2880 triangles.
 
+By default, Open3D tries to infer point cloud file type by extension. Below is
```

Comment:
I think "point cloud" should be changed to "meshes" in these sentences.

Reply:
Thanks!